features/bevragen/persoon/adressering/aanhef/autorisatie/autorisatie.feature

```diff
@@ -1,45 +1,54 @@
 # language: nl
 
-@gba @autorisatie
+@autorisatie
 Functionaliteit: autorisatie voor aanhef
 
   Rule: Vragen met fields om aanhef, wanneer de gebruiker niet geautoriseerd is voor alle velden waaruit aanhef wordt afgeleid, geeft een foutmelding
   
-    Scenario: Afnemer vraagt om aanhef en heeft uitsluitend de autorisatie die nodig is om deze vraag te mogen stellen
+    Abstract Scenario: Afnemer vraagt om aanhef en heeft uitsluitend de autorisatie die nodig is om deze vraag te mogen stellen
       Gegeven de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
-      | Rubrieknummer ad hoc (35.95.60)                                         | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
-      | 10120 10210 10220 10230 10240 10410 16110 50220 50230 50240 50610 50710 | N                        | 20201128                |
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 PANM03                    | N                        | 20201128                |
```

Comment:
PANM03 is rubrieknr voor aanhef, dus dan zou deze scenario niet moeten falen. Of is er nog wat anders waarop moet worden gecheckt?

Reply:
inderdaad, titel van scenario is immers "Afnemer (...) heeft uitsluitend de autorisatie die nodig is om deze vraag te mogen stellen". Dus mag het request geen foutmelding geven.
Moet zijn "Dan heeft de response 1 persoon"